internal_types/src/entry.rs

```diff
@@ -52,6 +52,16 @@ pub enum ColumnError {
 pub type Result<T, E = Error> = std::result::Result<T, E>;
 type ColumnResult<T, E = ColumnError> = std::result::Result<T, E>;
 
+// The flatbuffers structs for partition write, table write batch, and column
+// all have string values for their key/name. These values should always be
+// present. They are if the flatbuffer was generated by code here. However,
+// those are all returned as Option<&str> from the flatbuffer. So to keep us
+// from having option unwraps all over the place when dealing with this, we'll
+// use this default value to return in cases where the partition key, table
+// name, or column name are not there. If you see this value anywhere, you know
+// there's some illigitimate flatbuffer data somewhere.
+const UNKNOWN_STRING_VALUE: &str = "unknown_from_flatbuffer";
```

Comment:
```suggestion
pub const UNKNOWN_STRING_VALUE: &str = "unknown_from_flatbuffer";
```

This information (docs + the actual value) should be usable by users of `PartitionWrite::key`, `TableBatch::name`, etc.